cmd/utils/flags.go

```diff
@@ -995,6 +1005,7 @@ var (
 		DataDirFlag,
 		AncientFlag,
 		RemoteDBFlag,
+		HttpHeaderFlag,
```

Comment:
Although I feel like it's a bit weird to put this http header flag in the database group. But it's bound with `RemoteDBFlag` anyway, I guess it's fine.